### DISCUSSION TITLE
Custom Script with extra files

### DISCUSSION THREAD

**Discussion by kawalex** [netbox-community/netbox#17366] [Help Wanted!]
Hi, 
I have a custom script that uses TextFsm templates stored in others files.
When I add the new script in Netbox, I can select only one file while I need more than one (the .py file + the textfsm templates). Is there a way to manage this ?
Thanks and regards.
Alexandre.

**Comment by kawalex** [comment]
Any ideas ?

**Reply by zmaster7** [reply]
Imported scripts from file end up in the /opt/netbox/netbox/scripts/ directory. You can manually copy the additional files over to this directory to utilize them. Alternatively, if you setup a [data source](https://netboxlabs.com/docs/netbox/en/stable/models/core/datasource/) I believe the additional files would just have to be a part of the specified data source to use them.

**Reply by kawalex** [reply]
Thanks @zmaster7 for your inputs. I'm using data source, the script and the extra files are well seen in netbox, but when I add the script in Netbox WebUI, I can select only one file from the datasource and only this file is moved to scripts directory.
